src/components/ProductCard/ProductCart.tsx

```diff
@@ -1,14 +1,48 @@
+import styles from './productCart.module.css';
+import utils from '../../styles/modules/utils.module.css';
+import SelectedProduct from '../SelectedProduct/SelectedProduct';
+import StarIcon from '../Icon/StarIcon';
+
 interface ProductCardProps {
   image: string;
   title: string;
   price: number;
   description: string;
   rating: number;
-  isActive: boolean;
+  ratingNumbers: number;
 }
 
-const ProductCard = (_: ProductCardProps) => {
-  return <></>;
+const ProductCard = ({
+  image,
+  title,
+  price,
+  description,
+  rating,
+  ratingNumbers,
+}: ProductCardProps) => {
+  return (
+    <div className={styles.product}>
+      <div className={`${styles.imageGroup} ${utils.relative}`}>
+        <img className={styles.productImage} src={image} alt={title} />
+        <SelectedProduct />
+      </div>
+      <div className={styles.group}>
+        <div className={`${styles.nameGroup} ${utils.flexCenter}`}>
+          <p>{title}</p>
+          <span>{price}$</span>
+        </div>
+        <p className={styles.desc}>{description}</p>
+        <div className={`${styles.ratingGroup} ${utils.flexCenter}`}>
+          <div className={`${styles.numberRating} ${utils.flexCenter}`}>
+            {Array.from({ length: ratingNumbers }).map((_, index) => (
+              <StarIcon key={index} />
+            ))}
+          </div>
+          <p>{`(${rating}k)`}</p>
```

Comment:
what is `k` at the end

Reply:
"k" is the number of views evaluated by design

Reply:
that wrong the rating can be a positive integer that start from 0, when rating reach 1000 so that we shorthand into `1k`
but incase rating is 1, 11, or 999 should not have `k`

Reply:
fixed